bin/c-sdp-start.js

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const { exec } = require('shelljs');
+
+exec(
+    `minikube -p section start --disk-size 24g --extra-config=apiserver.service-node-port-range=80-32767 --kubernetes-version v1.16.2 --memory=4096 --vm-driver=vmware`
```

Comment:
@smebberson Should this use `c mk start -p section`?
I get this because the kubernetes version is set, is that for a particular reason?
```
! /usr/local/bin/kubectl is version 1.21.1, which may have incompatibilites with Kubernetes 1.16.2.
  - Want kubectl v1.16.2? Try 'minikube kubectl -- get pods -A'
```

Reply:
@allanchau, yeah, I'm pretty sure that's the best version of Kubernetes to use with Section devpop at the moment.